### ISSUE TITLE
Problem with Generic Scattering Calculator buttons (Trac #616)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#748]
Once a pdb file is loaded in the Generic Scattering Calculator window a new drop down combo box appears BEFORE the two Help and Close buttons, pushing them out of the viewable area.  However it does not force a redraw that allows the panel to realize it has elements off to the right so that a scroll bar can be added.  Two suggestions:
* fix so that the redraw alerts the parent that a bottom scroll bar is now required.
* move the Help and Close buttons one line down so they are not pushed to the right when the drop down box is added.

Migrated from http://trac.sasview.org/ticket/616
```json
{
    "status": "closed",
    "changetime": "2016-10-11T03:05:03",
    "_ts": "2016-10-11 03:05:03.580210+00:00",
    "description": "Once a pdb file is loaded in the Generic Scattering Calculator window a new drop down combo box appears BEFORE the two Help and Close buttons, pushing them out of the viewable area.  However it does not force a redraw that allows the panel to realize it has elements off to the right so that a scroll bar can be added.  Two suggestions:\n* fix so that the redraw alerts the parent that a bottom scroll bar is now required.\n* move the Help and Close buttons one line down so they are not pushed to the right when the drop down box is added.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView GUI Enhancements",
    "time": "2016-08-10T23:00:21",
    "component": "SasView",
    "summary": "Problem with Generic Scattering Calculator buttons",
    "priority": "minor",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "butler",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2016/09/13 13:57:19`: **butler** changed milestone from "SasView 4.0.0" to "SasView 4.1.0"

**Comment by butlerpd** [comment]
Trac update at `2016/10/11 03:05:03`: 
* **butler** commented:

In changeset 0f7c930fc2c325971c76a60d39f2e3719b8143a3:

```text
#!CommitTicketReference repository="sasview" revision="0f7c930fc2c325971c76a60d39f2e3719b8143a3"
force resize of panel (self.FitInside()) when the button sizer expands
due to user action.  Also puts buttons in two rows so not usually a
problem anyway now. Fixes #748
```


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "new" to "closed"
